daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -1634,9 +1634,24 @@ private[lf] object DecodeV1 {
       BuiltinFunctionInfo(DATE_TO_UNIX_DAYS, BDateToUnixDays),
       BuiltinFunctionInfo(EXPLODE_TEXT, BExplodeText),
       BuiltinFunctionInfo(IMPLODE_TEXT, BImplodeText),
-      BuiltinFunctionInfo(GEQ_DATE, BGreaterEq, implicitParameters = List(TDate)),
-      BuiltinFunctionInfo(LEQ_DATE, BLessEq, implicitParameters = List(TDate)),
-      BuiltinFunctionInfo(LESS_DATE, BLess, implicitParameters = List(TDate)),
+      BuiltinFunctionInfo(
+        GEQ_DATE,
+        BGreaterEq,
+        implicitParameters = List(TDate),
+        maxVersion = Option(genComparison),
```

Comment:
What is `Option`? Shouldn't that be `Some`?

Reply:
There is no reason to use Option.